frontend/src/features/admin-form/responses/common/utils/getPaymentDataView.ts

```diff
@@ -69,6 +69,14 @@ export const getPaymentDataView = (
       name: 'Payment amount',
       value: centsToDollarString(payment.amount),
     },
+    {
+      key: 'products',
+      name: 'Product/service',
+      value:
+        payment.products
+          ?.map(({ name, quantity }) => `${name} x ${quantity}`)
+          .join(', ') || '-',
```

Comment:
Note: Usage of `,` might cause the csv to insert `"` to wrap them. But any sufficiently robust code should be able to handle it properly. Tested using various spreadsheet tools, and they are able to handle the data properly as a string.

Excerpt of raw CSV
```
S$100.00,VIP x 1,"Tue, 1 Aug 2023, 01:46:52 AM",S$3.90,-,-
S$530.00,"VIP x 4, Regular x 2","Tue, 1 Aug 2023, 01:52:30 AM",S$7.44,-,-
```